src/bivot-js/bivot.js

```diff
@@ -2412,6 +2415,8 @@ class bivotJs {
     if (_self.mesh === null) {
       console.warn('Mesh unavailable; using planar geometry');
       _self.mesh = new THREE.Mesh(_self.getPlaneGeometry());
+      // Register for disposal
```

Comment:
Suggest a comment like "Although the planar mesh is never retrieved from the cache, store it anyway to dispose it on shutdown", to make clear the primary purpose of the cache is something other than disposal